users/tests.py

```diff
@@ -362,6 +362,63 @@ def test_updating_user_recreates_instance_cache(self):
         self.assertEqual(len(cache.keys(cache_key_name)), 1)
         self.assertEqual(cache.get(cache_key_name)["first_name"], first_name)
 
+    def test_user_can_update_its_own_settings(self):
+        test_settings = json.dumps(
+            {
+                "player": {"configuration": {"skipEnabled": False}},
+                "app": {"appearance": {"darkMode": True}},
+            }
+        )
+        # update settings for user 1
+        response = self.client.put(
+            f"/api/v1/users/{self.user.id}/setting/",
+            json.loads(test_settings),
+            format="json",
+        )
+
+        # 200 OK returned as status
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # The plio should contain the new updated settings object
```

Comment:
we have generally not including capital letters in general comment sentences
```suggestion
        # the plio should contain the new updated settings object
```